### PR TITLE
Documentation: add article on Introduction to Network Analysis

### PR DESCRIPTION
**Pull request by juliajuju93**
# Summary
- This branch adds an article **Introduction to Organizational Network Analysis with wpa in R** to the **pkgdown** website for this GitHub repo. 
- This article is not part of the installed package, and will only be featured on the website. 
- The article walks through the foundations of ONA, and the functions available in **wpa** to conduct the relevant analysis. 

# Changes
The changes made in this PR are:
1. Add `network-analysis.Rmd` to the `vignettes` subdirectory.
2. Update `.Rbuildignore` so that the article is not built as part of the installed package. 

# Checks
- [x] There are no spelling or grammar errors. 
- [x] All R CMD checks pass. 

# Notes
The next steps on this article would be:
- Create an article on advanced Network Analysis
- Update relevant content to the Analyst Guide Networks section.

### CODE REVIEW

**Comment by juliajuju93** [comment]
@martinctc - I have updated the rmd file -> let me know what you think :)

File: vignettes/network-analysis.Rmd [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
...imary aspects, a multitude of separate entities ("nodes") and the connections between them ("edges"). Nodes and edges are the foundation for understanding how information in your organization is flowing, can flow, and should flow. Nodes in ONA visualizations typically represent individual employees or groups of employees (e.g. `Organization`, `Region` etc.), serving as important hubs for the exchange of ideas and information. In addition to these nodes, you have connections (so-called "edges") between them. These edges indicate there is some relationship between the nodes, meaning these individuals or groups have a connection to each other when they collaborate. In the context of Workplace Analytics, we identify these connections between nodes in the network based on collaboration activities (emails, meetings, calls, instant messages). In a network analysis, we also need to understand the strength of the tie. For this calculation we are using time spent together in collaboration. In addition, we have to consider the **direction** in which collaboration is operated. For meetings there is no direction, but for emails, chats and calls we have a sense of direction which we can look at.
```
Comment (martinctc) [inline vignettes/network-analysis.Rmd:23]: @juliajuju93 The line on `is flowing, can flow, and should flow` sounds slightly excessive on what you can do from a graph. What a graph really tells you is how information _is flowing_, but the other two depends on factors that lie entirely outside the graph. Perhaps can substantiate this to say something alone the lines of, `nodes and edges provide a foundation on how information is flowing, and also strategize on how they can flow and should flow` ?

**Review by martinctc — CHANGES_REQUESTED** [review]
@juliajuju93 thank you for drafting up this article! I've made a few wording and formatting changes, but overall I think it is looking great. I've left a comment on this pull request so it would be great if you can have a look at that one. Another general comment is that it would be great to have an 'ending' paragraph to say something along the lines of, 'this is just an introduction but there is a lot more - reference the individual function documentation and examples if you want more details'.

After that, I think if you can fill out the pull request summary (on GitHub) and check that the plot images work, then this would be good for merging. Happy to review again once that's done - thank you so much!

**Comment by juliajuju93** [comment]
Hi @martinctc I have updated the last paragraph - feel free to review :)

**Review by martinctc — APPROVED** [review]

**Pull request merged**